packages/dashboard/src/components/Card/examples/MultiCountExample.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { ICardProps, CardSize, Priority, CardContentType } from '../Card.types';
+import { Card } from '../Card';
```

Comment:
Same, fixing imports here will help others use the examples better

Reply:
done